common/src/main/java/io/github/gaming32/bingo/util/BingoCodecs.java

```diff
@@ -1,29 +1,35 @@
 package io.github.gaming32.bingo.util;
 
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Sets;
 import com.mojang.datafixers.util.Either;
 import com.mojang.datafixers.util.Pair;
 import com.mojang.datafixers.util.Unit;
-import com.mojang.serialization.Codec;
-import com.mojang.serialization.DataResult;
-import com.mojang.serialization.Decoder;
-import com.mojang.serialization.DynamicOps;
+import com.mojang.serialization.*;
 import dev.architectury.registry.registries.Registrar;
-import io.github.gaming32.bingo.mixin.common.ContextAwarePredicateAccessor;
-import net.minecraft.advancements.critereon.ContextAwarePredicate;
 import net.minecraft.resources.ResourceKey;
 import net.minecraft.resources.ResourceLocation;
 import net.minecraft.util.ExtraCodecs;
-import net.minecraft.world.level.storage.loot.predicates.LootItemConditions;
 
+import java.util.Objects;
 import java.util.Optional;
+import java.util.Set;
+import java.util.function.Function;
 
 public final class BingoCodecs {
+    /**
+     * {@link DynamicOps} to use when you don't care about the type
+     */
+    public static final DynamicOps<?> DEFAULT_OPS = JsonOps.INSTANCE;
+    /**
+     * Empty {@link Dynamic} to use when you don't care about the ops
+     */
+    public static final Dynamic<?> DEFAULT_DYNAMIC = new Dynamic<>(DEFAULT_OPS);
```

Comment:
`EMPTY_DYNAMIC`

Reply:
I renamed it to this to match with `DEFAULT_OPS`, but maybe it should be the original name